menas/ui/service/EntityService.js

```diff
@@ -280,8 +280,28 @@ class DatasetService extends EntityService {
     return super.getByNameAndVersion(sName, iVersion, sModelPath, "#/dataset")
   }
 
+  cleanupEntity(oEntity) {
+    return {
+      description: oEntity.description,
```

Comment:
Could you reorder these to match the `scala` definition? It would make it easier to find if there are fields missing or unwanted.

Reply:
``` 
name,
version,
description,
hdfsPath,
hdfsPublishPath,
schemaName,
schemaVersion,
confromance
```